### ISSUE TITLE
Error closing IAEA phase space ID 44

### ISSUE THREAD

**Issue by ojalaj** [nrc-cnrc/EGSnrc#121]
When performing a DOSXYZnrc source 20 simulation, I get the above error printed in every `*.egslog` file of the synchronized phase space source simulation. Also, the lines above the message read:
```
 MIN PARTICLE WEIGHT FOR ALL SCORING ZONES =    0.0010000
 MAX PARTICLE WEIGHT FOR ALL SCORING ZONES =    0.0010000
```
which I think should be something else. Otherwise the simulation runs fine, although there is something wrong in the resulting dose distribution.

**Comment by ftessier** [comment]
@blakewalters can you track down the origin of the error message `Error closing IAEA phase space ID 44`

**Comment by blakewalters** [comment]
Does this appear in the .egslog file from the DOSXYZnrc run?  The 'MIN PARTICLE WEIGHT..' output you mention would only make sense from a BEAMnrc run.

**Comment by ojalaj** [comment]
Sorry for being unclear on this issue - the message was printed on the .egslog file of the shared library BEAM simulation run by DOSXYZnrc source 20 simulation. @blakewalters this resulted e.g. from the DOSXYZnrc source 20 simulation of which I have sent you the output by email related to #125

**Comment by ftessier** [comment]
Fixed in d70e11d.
